operator/bundle_helpers/README.md

```diff
@@ -1,27 +1,48 @@
 # Bundle Helpers
 
-## Python Dependency Management
-
 For hermetic builds with Konflux, we need to provide the full list of resolved dependencies in `requirements.txt`.
-These will be prefetched with Cachi2 and made available to the container image build.
+The dependency source files will be prefetched with Cachi2 and made available to the container image build.
+Because GHA/upstream is running on a different Python version, it uses `requirements-upstream.txt` to mange its own dependencies.
+Follow the procedure below after any dependencies change for successful builds in Konflux and mirror those changes into the `requirements-upstream.txt`.
```

Comment:
After looking at five `requirements*` files in the `bundle_helpers/` directory, I realized we should do better job explaining what is what, at least for four not-gitignored files. Do you mind adding a short list like?

* `requirements.in` -> blah
* `requirements.txt` -> blah
* `requirements-build.txt` -> blah
* ...